pkgs/development/libraries/libargon2/pkgconfig.patch

```diff
@@ -0,0 +1,23 @@
+diff --git a/libargon2.pc b/libargon2.pc
+index 8bce107..7ca636c 100644
+--- a/libargon2.pc
++++ b/libargon2.pc
+@@ -1,16 +1,10 @@
+-# libargon2 info for pkg-config
+-## Template for downstream installers:
+-## - replace @HOST_MULTIARCH@ with target arch, eg 'x86_64-linux-gnu'
+-## - replace @UPSTREAM_VER@ with current version, eg '20160406'
+-
+-prefix=/usr
+ exec_prefix=${prefix}
+-libdir=${prefix}/lib/@HOST_MULTIARCH@
++libdir=${prefix}/lib/
+ includedir=${prefix}/include
+ 
+ Name: libargon2
+ Description: Development libraries for libargon2
+-Version: @UPSTREAM_VER@
++Version: ${version}
```

Comment:
That doesn't seem to be done for this one.
Also, presumably I still need to patch the prefix=/usr to say prefix=@PREFIX@ or some such thing, right?